cms/Cargo.toml

```
@@ -14,29 +14,36 @@ edition = "2021"
 rust-version = "1.65"
 
 [dependencies]
-der = { version = "0.7.6", features = ["alloc", "derive", "oid", "pem"] }
+der = { version = "0.7.7", features = ["alloc", "derive", "oid", "pem"] }
 spki = { version = "0.7" }
 x509-cert = { version = "0.2.3", default-features = false, features = ["pem"] }
 const-oid = { version = "0.9", features = ["db"] } # TODO: path = "../const-oid"
```

```suggestion
const-oid = { version = "0.9.4", features = ["db"] }
```

This would pull any `const-oid <0.10 >= 0.9.4`

---

I'm surprised minimal-version CI job  doesn't complain here though.

---

It's possible a (transitive) dependency already has this locked

---

I don't believe this is the case though.

As far as I can tell it pulls `0.9.2`:
```
[[package]]
name = "const-oid"
version = "0.9.2"
```

---

well because we sill rely on the duplicated definition ^^ I should learn how to read a diff sometimes.